airflow/contrib/hooks/aws_hook.py

```diff
@@ -30,7 +30,10 @@
 
 
 class AwsHook(AwsBaseHook):
-    """This class is deprecated. Please use `airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`."""
+    """
+    This class is deprecated.
+    Please use `airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`.
```

Comment:
```suggestion
    Please use :class:`airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`.
```

I think this should work? Same for other deprecated classes below.